src/cli/commands.js

```diff
@@ -3,14 +3,28 @@
 module.exports = {
   usage: 'Usage: $0 <command> [options]',
   commands: {
+    clean: {
+      cmd: 'clean <dirPath>',
+      example: { cmd: '$0 clean components --yes' },
+      description: `Remove the node_modules directory from each component's subfolder`,
+      options: {
+        yes: {
+          boolean: true,
+          description: 'Skip all confirmation prompts',
+          default: false
+        }
+      },
+      usage: 'Usage: $0 clean <dirPath>'
+    },
+
     dev: {
       cmd: 'dev <dirPath> [port] [baseUrl]',
       example: {
         cmd:
           '$0 dev ../all-components 3001 127.0.0.1:3001 --fallbackRegistryUrl=http://anotherhost:anotherport/'
       },
       description:
-        'Runs a local oc test registry in order to develop and test components',
+        'Run a local oc test registry in order to develop and test components',
```

Comment:
Took inspiration from lerna here, all the actions are intended as 2nd person rather than 3rd person intending the action is done by the user not the CLI really :)